modules/installation-special-config-mirrored-disk.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-customizing.adoc
+
+[id="installation-special-config-mirrored-disk_{context}"]
+= Mirroring disks during installation
+
+During {product-title} installation, you can enable mirroring of the boot disk to two or more redundant replicas on master and worker nodes. A node stays active after failure as long as one replica still functions. Mirroring support is available on x86_64 nodes booted with BIOS or UEFI and on ppc64le nodes.
+
+Mirroring does not support replacement of a failed disk. To restore the mirror to a pristine, non-degraded state, reprovision the node. For disk encryption, an alternate method of configuring disk encryption is required.
+
+[IMPORTANT]
+====
+Mirroring is available only for user-provisioned infrastructure deployments on {op-system-first} systems.
+====
+
+.Procedure
+
+To enable boot disk mirroring during {product-title} deployment:
+
+. Follow the bare metal install procedure up to the point where you created Ignition configs for your installation.
++
+For example, you should have entered the following command:
++
+[source,terminal]
+----
+$ openshift-install create ignition-configs --dir $HOME/clusterconfig
+----
++
+. Verify that the Ignition config files were created:
++
+[source,terminal]
+----
+$ ls $HOME/clusterconfig/
+----
++
+[source,terminal]
+.Example output
+----
+auth  bootstrap.ign  master.ign  metadata.json  worker.ign
+----
+
+. Create a {op-system} Config (RCC) with a reference to your `master.ign` or `worker.ign` config, depending on the type of node you are configuring. The RCC must specify the devices to be used for boot disk mirroring. If you want, you can configure LUKS encryption for the root filesystem.
++
+The following Ignition config example demonstrates how to create `$HOME/clusterconfig/worker-raid.rcc`:
++
+[source,yaml]
+.Ignition config example
+----
+variant: rhcos
+version: 0.1.0
+ignition:
+  config:
+    merge:
+      - local: worker.ign <1>
+boot_device:
+  mirror:
+<<<<<<< HEAD
+    devices: <2>
+      - /dev/sda
+      - /dev/sdb
+  luks: <3>
+    tpm2: true <4>
+    tang: <5>
+      - url: https://tang.example.com
+        thumbprint: <tang_thumbprint>
+----
++
+<1> Use the name of the Ignition config for your node type.
+<2> List all disk devices that should be included in the boot disk mirror, including the disk that {op-system} will be installed onto.
+<3> Include this section if you want to encrypt the root filesystem. See "Encrypting disks during installation" for more details.
+<4> Include this field if you want to use a Trusted Platform Module (TPM) to encrypt the root filesystem. See "Enabling TPM v2 disk encryption" for more details.
+<5> Use a Tang server to encrypt the root filesystem. To obtain the server URL and thumbprint, follow the instructions in "Enabling Tang disk encryption".
```

Comment:
For consistency, let's phrase this "Include this section if you want to use a Tang server..."